docs/tutorial/node/src/getting_started.md

```diff
@@ -0,0 +1,44 @@
+# Getting Started
+
+In this tutorial, we use JavaScript as the host programming language. We also support other programming languages like
+[Rust](https://slint-ui.com/docs/rust/slint/) or [C++](https://slint-ui.com/docs/cpp/).
+
+You will need a development environment with [node.js 16](https://nodejs.org/download/release/v16.19.1/) and [npm](https://www.npmjs.com/) installed.
```

Comment:
maybe mention that more recent version are not supported (with link to the github issue about new neon version)

Reply:
done